src/v/redpanda/admin_server.cc

```diff
@@ -2017,4 +2029,71 @@ void admin_server::register_transaction_routes() {
 
           co_return ss::json::json_return_type(ans);
       });
+
+    ss::httpd::transaction_json::delete_partition.set(
+      _server._routes,
+      [this](std::unique_ptr<ss::httpd::request> req)
+        -> ss::future<ss::json::json_return_type> {
+          auto transaction_id = req->param["transactional_id"];
+
+          auto namespace_from_req = req->get_query_param("namespace");
+          auto topic_from_req = req->get_query_param("topic");
```

Comment:
do these need validated? also, most of our APIs put the ntp in the URL path rather than as query parameters. Should we do that here too?

Reply:
but if the parameter is missing, then the default value is `""` which will eventually fail because the resulting ntp won't be found, but the user should have received an error that they didn't provide the required parameters.

however, on further inspection, it looks like the JSON specification you wrote marked the parameters as required. in this case I *think* seastar will provide that validation automatically so this seems ok :)

Reply:
Oh, you wrote about is it empty or not. 
Yes, I use required flag in json for it. But, I wanna check does it work or not

Reply:
Vadim made a right call here, it's true that ntp usually is a part of the path because its the object the api work with; but in this case transaction is the object and we instruct it to remove a particular ntp